products/couchbase-server.md

```diff
@@ -22,17 +22,17 @@ releases:
 -   releaseCycle: "7.1"
     eol: false
     latest: "7.1.3"
-    latestReleaseDate: 2023-01-31
+    latestReleaseDate: 2023-02-01
```

Comment:
Same as https://github.com/endoflife-date/endoflife.date/pull/2429#discussion_r1095514854, date updated without the version.

Reply:
Explanation : https://github.com/endoflife-date/talk/wiki/couchbase-server.